Doc/c-api/call.rst

```diff
@@ -320,18 +333,20 @@ please see individual documentation for details.
    *NULL* on failure.
 
 
-.. c:function:: PyObject* _PyObject_CallMethodNoArgs(PyObject *obj, PyObject *name)
+.. c:function:: PyObject* PyObject_CallMethodNoArgs(PyObject *obj, PyObject *name)
 
    Call a method of the Python object *obj* without arguments,
    where the name of the method is given as a Python string object in *name*.
 
    Return the result of the call on success, or raise an exception and return
    *NULL* on failure.
 
+   This function is not part of the `limited API <stable>`_.
```

Comment:
Maybe put it also in the limited API. I don't see any issue with the stable ABI.

Reply:
Can we do it in a separate commit?
I know you wanted to change the API to add `tstate`; let's revisit the stable ABI after that decision is made?

Reply:
Ok, ignore my comment: it can be discussed after this PR is merged.